frontend/control-center/src/pages/Connectors/ConfigStatusButton/index.tsx

```diff
@@ -12,12 +14,33 @@ interface ConfigStatusButtonProps {
 export const ConfigStatusButton = (props: ConfigStatusButtonProps) => {
   const {componentStatus, customStyle} = props;
   const {t} = useTranslation();
+  const navigate = useNavigate();
+
+  const handleNavigation = (componentStatus: ComponentStatus, event: Event) => {
+    switch (componentStatus) {
+      case ComponentStatus.notHealthy:
+        event.stopPropagation();
+        navigate(STATUS_ROUTE);
+        break;
+      case ComponentStatus.notConfigured:
+        event.stopPropagation();
+        console.log('navigate to configurationComponent');
```

Comment:
nitpicking: I find it a bit overkill to make a switch case only for 2 cases, but of course, it is perfectly valid 👍 

💣 console.log line 27